### DISCUSSION TITLE
NGINX Variation

### DISCUSSION THREAD

**Discussion by moisish** [serversideup/docker-php#377] [Ideas & Feature Requests]
It will be a nice idea if we could have an nginx image variation without the fpm.

We will be able to run nginx and php-fpm in different containers which can achieve more things like scaling of the fpm container, different container for queues, or even host multiple sites with the same nginx container but on different php versions

**Comment by jaydrogers** [comment]
I could see this being valuable, but I would have to put it under another project since running NGINX only is out of scope of a PHP image (example: `serversideup/php:nginx`)

If there gets a lot of traction on this, I could try and maintain a separate project for it.

**Reply by moisish** [reply]
Indeed it does not fit straight under PHP but at the same time it is linked for the reasons I mentioned above. I will be happy to help if needed.

Sidenote, will it be possible the current nginx,apache images support multisites?

**Reply by jaydrogers** [reply]
> Sidenote, will it be possible the current nginx,apache images support multisites?

We run one container per website for security reasons.

If you'd like to add multiple sites under one, you'd have to set `DISABLE_DEFAULT_CONFIG=true` then provide your own configs.

**Reply by moisish** [reply]
Do you use traefik in front of nginx container to route the traffic to the correct container?

**Reply by jaydrogers** [reply]
Yup, that's exactly what we do!

Traefik handles all the SSL and routing.
